code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -40,6 +40,24 @@
 	resistance_flags = NONE
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 80, "acid" = 50)
 
+/obj/item/clothing/gloves/combat/plus
+	name = "combat gloves plus"
+	desc = "These tactical gloves are fireproof and shock resistant,and using a nanochip technology it teaches you the powers of krav maga."

```

Comment:
resistanct,and -> resistant, and

Reply:
i have good england (i'll fix it in a second.)